src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -221,6 +221,7 @@ def load_arguments(self, _):
 
     with self.argument_context('group create') as c:
         c.argument('rg_name', options_list=['--name', '--resource-group', '-n', '-g'], help='name of the new resource group', completer=None)
+        c.argument('managed_by', min_api='2016-09-01', help='The ID of the resource that manages this resource group.', resource_type=ResourceType.MGMT_RESOURCE_RESOURCES)
```

Comment:
You don't need to specify the resource_type here. It already registered when module initialization.

Reply:
OK, I'll make it registered with the initialized __init__.py